k8_cortx_cloud/deploy-cortx-cloud.sh

```diff
@@ -1325,4 +1320,4 @@ if [[ $num_motr_client -gt 0 ]]; then
     deployCortxClient
 fi
 deployCortxServices
-cleanup
```

Comment:
How should I think about this?  Is this (unintentional) change a problem?

Reply:
We need that cleanup step to be there, so you'll want to add it back in.

Reply:
It wasn't clear from this diff, but the difference is that a newline was removed in my change at the end of the file (this is the last line in the file)